wiki/Ranking_Criteria/Difficulty_Naming/en.md

```diff
@@ -0,0 +1,139 @@
+# Difficulty Naming
+
+This wiki article serves as a substitute and an update to [this topic](https://osu.ppy.sh/community/forums/topics/178700). 
+
+Difficulty naming is a setting in the song setup that creators are allowed to change to best represent their beatmap's difficulty. As there are many difficulty names, this article contains the ways in which difficulties in each respective game mode are commonly named, as well as various other alternatives that are also allowed.
```

Comment:
"The purpose of difficulty naming is telling players how difficult a map is. The scheme for this is up to creators, this article lists common conventions as well as their contexts"

Reply:
also should mention that these are just the most common conventions and make it more clear that these aren't the only names that can be used - as long as the set of difficulty names is descriptive it's fine ^^^^^^

especially the last part of the sentence makes it seem like "other variations that are also allowed" = "other variations are not allowed"